website/templates/profile/account.mako

```diff
@@ -5,39 +5,101 @@
+                                    <td style="width:100%">{{ $data.address }}</td>
+                                    <td><a data-bind="click: $parent.makeEmailPrimary">make&nbsp;primary</a></td>
+                                    <td><a data-bind="click: $parent.removeEmail"><i class="fa fa-times text-danger"></i></a></td>
+                                </tr>
+                            </tbody>
+                        </table>
+
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th colspan="2">Unconfirmed Emails</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <!-- ko foreach: unconfirmedEmails() -->
+                                <tr>
+                                    <td style="width:100%">{{ $data.address }}</td>
+                                    <td><a data-bind="click: $parent.removeEmail"><i class="fa fa-times text-danger"></i></a></td>
+                                </tr>
+                                <!-- /ko -->
+                                <tr>
+                                    <td colspan="2">
+                                        <form data-bind="submit: addEmail">
+                                            <div class="form-group">
+                                                <input data-bind="value: emailInput" class="form-control" name="addEmail">
+                                            </div>
+                                        <input type="submit" value="Add Email" class="btn btn-default">
+                                        </form>
+                                    </td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+                <div id="changePassword" class="panel panel-default">
+                    <div class="panel-heading"><h3 class="panel-title">Change Password</h3></div>
+                    <div class="panel-body">
+                        <form id="changePasswordForm" role="form" action="${ web_url_for('user_account_password') }" method="post">
+                            <div class="form-group">
+                                <label for="old-password">Old password</label>
+                                <input type="password" class="form-control" name="old_password">
+                            </div>
+                            <div class="form-group">
+                                <label for="password">New password</label>
+                                <input type="password" class="form-control" name="new_password">
+                            </div>
+                            <div class="form-group">
+                                <label for="password2">Confirm new password</label>
```

Comment:
Since you're providing the `for` attribute, it would make sense to add the `name` attribute to the `input` tags.